wlc/__init__.py

```diff
@@ -512,6 +518,15 @@ def list(self):
         self.ensure_loaded("translations_url")
         return self.weblate.list_translations(self._attribs["translations_url"])
 
+    def add_translation(self, language):
```

Comment:
why don't we call this `add_language` to make it more clear what it does?

Reply:
Because "language" is a different concept in the weblate api (you can add custom languages with custom language codes). In weblate parlance, a "translation" is essentially a join between a language and a component. Translations contain all of the `units`, and each unit has a "source" and a "target" (along with other data). The `unit` is what we would normally call a `translation` when speaking normally ("Hoi is the dutch translation of Hello"). I think it makes sense to use `add_language` in our dragoman-core code, but since this will ideally get merged into the official wlc, we should stick to their terminology.